src/analysis/Environments/Environments.ts

```diff
@@ -180,24 +178,18 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
       });
     }
 
-    const decorateLabeledResourceWithWorkspace = <T extends ListRuntimeItem | PersistentDisk | App>(
-      cloudObject: T
-    ): DecoratedResourceAttributes & T => {
-      const {
-        labels: { saturnWorkspaceNamespace, saturnWorkspaceName },
-      } = cloudObject;
-      const { workspace } = workspaces[saturnWorkspaceNamespace]?.[saturnWorkspaceName] || {};
-      // Attempting to catch resources related to GCP v1 workspaces (Rawls no longer returns them).
+    const decorateWithWorkspace = (cloudObject) => {
+      const googleProject = cloudObject.googleProject || cloudObject.cloudContext?.cloudResource;
```

Comment:
Could this just be `cloudObject.cloudContext?.cloudResource`? Only runtimes have a `googleProject` field